data-analysis.js

```diff
@@ -0,0 +1,76 @@
+const url = 'https://dummyjson.com/users';
+getData(url)
+	.then(users => {
+		const results = processData(users);
+		displayResults(results);
+	})
+
+function getData(url) {
+	return fetch(url)
+		.then(response => response.json())
+		.then(data => {
+		return data.users;
+		})
+	}
+
+function processData(users) {
```

Comment:
You can split this function into multiple function like getMinWeightUser, getAverageHeight, ...

Reply:
I can try to do this in the next branch when I make the next pull request

Reply:
Okay, approved